jako/distribute/RemoteScan.py

```diff
@@ -2,7 +2,6 @@
 
 
 class RemoteScan(Scan):
-    
     def __init__(self,
```

Comment:
I'm wondering if it would be better to handle the `Scan` params as kwargs, because otherwise you always have to update it in several places. I think it makes sense to have it for `DistributedScan` because it is something that user will directly interact with. But because RemoteScan is not, then maybe better just use kwargs. What do you think?

Reply:
I think that would be a good idea.

Reply:
Let's do that then.

Reply:
This is done in the latest commit